app/validator/element.ts

```diff
@@ -1,71 +1,71 @@
-const _ = require('lodash');
-const sdk = require('matrix-js-sdk');
-const config = require('app/config');
-const logger = require('app/logger');
-const Validator = require('app/validator/base');
-const { ValidatorEvent } = require('app/validator/events');
-const { RequestJudgementCollection, RiotCollection } = require('app/db');
-
-const utils = require('app/utils');
+import _ from 'lodash';
+import { createClient, MatrixClient, Room } from 'matrix-js-sdk';
+import config from 'app/config';
+import logger from 'app/logger';
+import Validator from './base';
+import { ValidatorEvent } from 'app/validator/events';
+import { RequestJudgementCollection, RiotCollection } from 'app/db';
+import { createJwtToken } from 'app/utils';
+import Config from 'types/config';
 
 const CHAIN_NAME = config.chain.name || '';
 
 class ElementValidator extends Validator {
-    constructor(config) {
+    private readonly chainName = 'kusama';
+    private readonly maxRetries = 10;
+    private readonly pollingRoomMessageInterval = 3;
```

Comment:
it seems this parameter `pollingRoomMessageInterval` never used anymore. 
We can safely remove it